packages/suite-native-navigation/src/TabBar.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { TouchableOpacity, View } from 'react-native';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { BottomTabBarProps } from '@react-navigation/bottom-tabs';
+import { Icon, IconType } from '@trezor/atoms';
+import { RouteTabs } from './routes';
+
+const tabBarStyle = prepareNativeStyle(utils => ({
+    height: 86,
+    backgroundColor: utils.colors.gray100,
+    borderTopColor: utils.colors.gray300,
+    borderTopWidth: utils.borders.widths.sm,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    paddingTop: 20.5,
+    paddingLeft: 51.5,
+    paddingRight: 51.5,
+}));
+
+export const bottomTabOptions: Record<string, { icon: IconType }> = {
+    [RouteTabs.Home]: { icon: 'home' },
+    [RouteTabs.Accounts]: { icon: 'accounts' },
+    [RouteTabs.Action]: { icon: 'action' },
+    [RouteTabs.Prices]: { icon: 'prices' },
+    [RouteTabs.SettingsStack]: { icon: 'settings' },
+};
+
+export const TabBar = ({ state, descriptors, navigation }: BottomTabBarProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <View style={[applyStyle(tabBarStyle)]}>
+            {state.routes.map((route, index) => {
+                const { options } = descriptors[route.key];
+                const isFocused = state.index === index;
+
+                const handleTabPress = () => {
+                    const event = navigation.emit({
+                        type: 'tabPress',
+                        target: route.key,
+                        canPreventDefault: true,
+                    });
+
+                    if (!isFocused && !event.defaultPrevented) {
+                        // The `merge: true` option makes sure that the params inside the tab screen are preserved
+                        navigation.navigate(route.name, { merge: true });
+                    }
+                };
+
+                return (
+                    <TouchableOpacity
+                        accessibilityRole="button"
+                        accessibilityState={isFocused ? { selected: true } : {}}
+                        accessibilityLabel={options.tabBarAccessibilityLabel}
+                        onPress={() => {
+                            // Calling in a function prevents Animation warning in iOS simulator (see https://github.com/react-navigation/react-navigation/issues/7839#issuecomment-829438793 for more details)
+                            handleTabPress();
+                        }}
+                        key={route.key}
+                    >
+                        <Icon
+                            type={bottomTabOptions[route.name].icon}
+                            size="big"
+                            color={isFocused ? 'black' : 'gray500'}
+                        />
+                    </TouchableOpacity>
```

Comment:
Do you mean to create something like TabBarIcon and move it to the atoms package? Is there a true reason for this? I guess it won't be reusable at all anywhere except this one and only tab bar.

Reply:
To me it seems like this could be a TapBarList (this cmp) + TapBar cmp which would be something like 
```
state.routes.map((route, index) => {
    // some code, destructuring etc
    return <TabBar {...props} />
})
```

It's not really going to be that much about reusability but imho the render method would be much more cleaner. In this component everything about the general tabs will be very clear and then you have a specific component of <TabBar />. My OCD likes this approach much more to have mapped items as a standalone component but that's just my view. Maybe you prefer to have it all inside one component like this and I just need to supress my OCD feelings 😀

Reply:
Ok, I understand your thoughts. I cleaned it up a little in https://github.com/trezor/trezor-suite/pull/5455/commits/586add7a095903bc6b92f49f63f61c3c7927d312